santos_akishagelsey/country/countries.css

```diff
@@ -1,46 +1,48 @@
 body {
-    font-family: Arial, sans-serif;
-    margin: 20px;
-    background-color: #f0f0f0;
+  font-family: Arial, sans-serif;
+  margin: 20px;
+  background-color: #f0f0f0;
 }
 
 h1 {
-    color: #333;
+  color: #333;
 }
 
 input[type="text"] {
-    width: 200px;
-    padding: 10px;
-    margin-right: 10px;
-    border: 1px solid #ccc;
-    border-radius: 5px;
+  width: 200px;
+  padding: 10px;
+  margin-right: 10px;
+  border: 1px solid #ccc;
+  border-radius: 5px;
 }
 
 button {
-    padding: 10px 20px;
-    border: none;
-    border-radius: 5px;
-    background-color: #4CAF50;
-    color: white;
-    cursor: pointer;
+  padding: 10px 20px;
+  border: none;
+  border-radius: 5px;
+  background-color: #4caf50;
+  color: white;
+  cursor: pointer;
 }
 
 button:hover {
-    background-color: #45a049;
+  background-color: #45a049;
 }
 
-#countryDetails, #countriesInRegion {
-    margin-top: 20px;
-    padding: 20px;
-    border: 1px solid #ccc;
-    border-radius: 5px;
-    background-color: #fff;
+#countryDetails,
+#countriesInRegion {
+  margin-top: 20px;
+  padding: 20px;
+  border: 1px solid #ccc;
+  border-radius: 5px;
+  background-color: #fff;
 }
 
-#countryDetails h2, #countriesInRegion h2 {
-    color: #333;
+#countryDetails h2,
+#countriesInRegion h2 {
```

Comment:
IDs should be written in snake case. All lower case separated by underscore.

Reply:
"countryDetails", "countriesInRegion" is not mentioned in countries.html
change in into:
"countryDetails" -> "country_details" 
"countriesInRegion" -> "countries_in_region"